prac_05/emails.py

```diff
@@ -0,0 +1,15 @@
+def main():
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        parts = email.split("@")
+        parts = parts[0].split(".")
+        name = " ".join(parts).title()
+        name_confirmation = input(f" Is your name {name}? (Y/N)").upper()
+        if name_confirmation == "N":
```

Comment:
I don't think this condition is sufficient, based on the sample output. ("Y" or "" for yes, everything else for no)